cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceContextConfig.java

```diff
@@ -265,7 +265,7 @@ private void setConf(CConfiguration conf) {
   }
 
   public CConfiguration getConf() {
-    CConfiguration conf = CConfiguration.create();
+    CConfiguration conf = CConfiguration.createEmpty();
```

Comment:
Added static methods. Using URL as a parameter for the create to make it in consistent with the `addResource` method parameters. Please let me know if this is ok.